content/en/serverless/step_functions/link-with-lambda-traces.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Link Step Functions and Lambda Traces
```

Comment:
minor: The keyword `Link` might give wrong impression as if they are still going to be 2 traces and linked via span linked or some references. Maybe `Merge` is better?